user-processing.js

```diff
@@ -0,0 +1,3 @@
+export function getUserValues(userObj){
+    return Object.values(userObj).join("|")
+}
```

Comment:
This is a good function - but maybe we could try making it an arrow function?
```suggestion
export const getUserValues = (userObj) => Object.values(userObj).join("|")
```